Essentials/src/messages.properties

```
@@ -182,6 +182,7 @@ holdFirework=\u00a74You must be holding a firework to add effects.
 holdPotion=\u00a74You must be holding a potion to apply effects to it.
 holeInFloor=\u00a74Hole in floor\!
 homes=\u00a76Homes\:\u00a7r {0}
+homeConfirmation=\u00a76You already have a home named \u00a7c{0}\u00a76!\nType the command again to confirm replacement.
```

Remain consistent style with other confirm messages

```suggestion
homeConfirmation=\u00a77To \u00a7lCONFIRM\u00a77 overwriting your existing home, \u00a7c{0}\u00a77, please repeat the command.
```

---

If anything, the style and format of that message varies wildly from the rest of Essentials. I'm not saying the message in this PR is perfect, but I think that one is also a bit far off the mark.

---

```suggestion
homeConfirmation=\u00a76You already have a home named \u00a7c{0}\u00a76!\nTo overwrite your existing home, type the command again.
```